lib/debug-server.js

```diff
@@ -62,8 +63,12 @@ DebugServer.prototype._getDebuggerPort = function(url) {
   return parseInt((/\?port=(\d+)/.exec(url) || [null, this._config.debugPort])[1], 10);
 };
 
-DebugServer.prototype._createSession = function(debugPort) {
-  return Session.create(debugPort, this._config);
+DebugServer.prototype._getDebuggerHost = function(url) {
+  return (/[\?\&]host=([0-9.a-zA-Z:/]+)/.exec(url) || [null, this._config.debugHost])[1];
```

Comment:
This will not support `?host=my-app.example.com` or even `?host=foo%20bar.com` (although I am not sure if a hostname is allowed to contain a character that needs escaping).


Here is the best solution to use:

```js
// at the top:
var url = require('url');
var querystring = require('querystring');

// here:
DebugServer.prototype._getDebuggerHost = function(urlString) {
  var parsedUrl = url.parse(urlString);
  var query = querystring.parse(parsedUrl.query);
  return query.host;
}
```